packages/now-next/src/index.ts

```diff
@@ -1156,37 +1162,14 @@ export const build = async ({
         },
         continue: true,
       },
-      { src: path.join('/', entryDirectory, '_next(?!/data(?:/|$))(?:/.*)?') },
-
-      // Next.js page lambdas, `static/` folder, reserved assets, and `public/`
-      // folder
-      { handle: 'filesystem' },
-
-      // This needs to come directly after handle: filesystem to make sure to
-      // 404 and clear the cache header for _next requests
-      {
-        src: path.join(
-          '/',
-          entryDirectory,
-          '_next/static/(?:[^/]+/pages|chunks|runtime|css|media)/.+'
```

Comment:
We still need this after handle: miss

Reply:
> Since it's only applied during `handle: hit` it would never need to be cleared in `handle: miss` since it can't `miss` after `hit` right?

ditto here